target/linux/ipq806x/files-5.10/arch/arm/boot/dts/qcom-ipq8065-ac400i.dts

```diff
@@ -0,0 +1,338 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qcom-ipq8065.dtsi"
```

Comment:
this should use the smb variant

Reply:
Marking as resolved, since I dropped 5.10 changes.

Reply:
this also apply to 5.15

Reply:
Done.